.github/workflows/test-publish.yml

```diff
@@ -0,0 +1,20 @@
+name: Test publish
+
+on:
+  workflow_call:
+
+jobs:
+  test-publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Setup
+        uses: ./.github/actions/setup
+        with:
+          enable-build-cache: 'true'
+      # TODO use retrieve artifact instead of build
+      - name: Build
+        run: yarn build
+      # TODO check jest cache
```

Comment:
Not really needed actually.
The cache of jest will not really help to run the ITs :S